packages/cli/src/utils/input.ts

```diff
@@ -1,22 +1,22 @@
-export function parseArgs(args) {
+export function parseArgs(args: string): string[] | never {
```

Comment:
Yeah, I thought about typing that as `any` instead of `string`. However, when I saw line 2, I thought "ok, the code expects this to be a string". I decided to leave it as is, because like this, if the user is using Typescript, the error will be generated statically, and if the user is using Javascript, the error will be generated dynamically.